templates/source/access-points.html

```diff
@@ -117,9 +117,13 @@
                 </div>
             </div>
             <div class="form-group">
-                <div class="col-sm-12">
+                <div class="col-sm-12 text-right">
                     <br />
-                    <button type="submit" class="btn btn-success pull-right">{% trans "Save" %}  <i class="fa fa-check"></i></button>
+                    <a href="{% url 'add-access-point' source.uuid %}" class="btn btn-danger">
```

Comment:
One final change here -- we want `add-access-point` to be `view-access-point`:

```suggestion
                    <a href="{% url 'view-access-point' source.uuid %}" class="btn btn-danger">
```